### PR TITLE
Add clarification comments to gruntfile

### PR DESCRIPTION
**Pull request by Shaderpixel**
Added comments to clarify RequireJS config object and related variables.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -384,7 +393,7 @@ module.exports = function (grunt) {
 		watch: {
 			sass: {
 				files: [templateScss + '/**/*.scss', templateJs + '/js-css/*.scss'],
-				tasks: ['scssGlobber','sass']
+				tasks: ['sassGlobber:all','sass']
```
Comment (mfairchild365) [inline Gruntfile.js]: This doesn't appear to be a documentation change. What is the purpose of this change?

Comment (Shaderpixel) [reply Gruntfile.js]: Just making the task scope be more specific and only run the 'all' task defined under scssGlobber instead of running the entire scssGlobber task (at this time there's only 'all' subtask, future there could be others).
